### PR TITLE
mixer: minor doc fixes

### PR DESCRIPTION
**Pull request by kyessenov**
- fix proto docs for kubernetesenv
- mark mixs crd command deprecated

Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: mixer/adapter/kubernetesenv/config/config.proto [hunk 1]
```diff
@@ -69,15 +69,15 @@ message Params {
...
+    // Namespace of the secret created for multi-cluster support.
     //
-	// Details on multi-cluster and the kubernetes secret required to
-	// access the remote cluster's credentials can be found here:
-	// https://istio.io/docs/setup/kubernetes/multicluster-install
-	//
-    // NOTE: If cluster_registries_namespace is not set then the environmental
-    // variable POD_NAMESPACE will be checked/used. If POD_NAMESPACE is not
-    // set then cluster_registries_namespace will default to "istio-system".
+    // Details on multi-cluster and the kubernetes secret required to
```
Comment (geeknoid) [inline mixer/adapter/kubernetesenv/config/config.proto]: multicluster, no dash.

File: mixer/adapter/kubernetesenv/config/config.proto [hunk 2]
```diff
@@ -69,15 +69,15 @@ message Params {
...
+    // access the remote cluster's credentials can be found here:
+    // <https://istio.io/docs/setup/kubernetes/multicluster-install>
```
Comment (geeknoid) [inline mixer/adapter/kubernetesenv/config/config.proto]: Please make this a markdown link `[Multicluster Install](....)`

File: mixer/adapter/kubernetesenv/config/config.proto (hunk 1 above)
Comment (kyessenov) [reply mixer/adapter/kubernetesenv/config/config.proto]: Done

File: mixer/adapter/kubernetesenv/config/config.proto (hunk 2 above)
Comment (kyessenov) [reply mixer/adapter/kubernetesenv/config/config.proto]: Done

**Comment by kyessenov** [comment]
@duderino please merge this to improve our docs.

**Comment by kyessenov** [comment]
@wenchenglu please take a look.

**Pull request merged**